### PR TITLE
Add more information and context to event pipelines docs

### PR DESCRIPTION
**Pull request by estherk15**
### What does this PR do? What is the motivation?

- Add more information and context to Event Management Pipelines
- Add a further reading

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/service_management/events/pipelines_and_processors/_index.md [hunk 1]
```diff
@@ -1,42 +1,65 @@
...
 
-### Common Uses Cases 
-- Enrich your events with additonal information from your CMDB
-- Standardize the tags on the events
-- Create new tags from the content within the events 
+Create an Pipeline to filter to the events that your interested in, for example, a source or a tag. 
 
+1. Navigate to [Event Management Pipelines][1] in the Datadog app.
+1. Click **Add a Pipeline**.
+1. Choose a filter from the dropdown menu or create your own filter query in the [Event Management Explorer][2] by selecting the `</>` icon. Filters let you limit what kinds of events a pipeline applies to. **Note**: The pipeline filtering is applied before any of the pipeline’s processors. You cannot filter on an attribute that is extracted in the pipeline itself.
```
Comment (estherk15) [inline content/en/service_management/events/pipelines_and_processors/_index.md:32]:
```suggestion
1. Choose a filter from the dropdown menu or create your own filter query in the [Event Management Explorer][2] by selecting the `</>` icon. Filters let you limit what kinds of events a pipeline applies to. **Note**: The pipeline filtering is applied before any of the pipeline's processors. You cannot filter on an attribute that is extracted in the pipeline itself.
```

[the remark this answers is not in the document]
Comment (estherk15) [reply content/en/service_management/events/pipelines_and_processors/_index.md:32]: Yes! I agree, that's correct. Is this clearer: "Use filters to limit/apply the pipeline processors to specific events" ?
Either limit or apply

File: content/en/service_management/events/pipelines_and_processors/_index.md [hunk 2]
```diff
@@ -1,42 +1,65 @@
...
 
-### Common Uses Cases 
-- Enrich your events with additonal information from your CMDB
-- Standardize the tags on the events
-- Create new tags from the content within the events 
+Create an Pipeline to filter to the events that you are interested in, for example, a source or a tag. 
 
+1. Navigate to [Event Management Pipelines][1] in Datadog.
+1. Click **Add a Pipeline**.
+1. Choose a filter from the dropdown menu or create your own filter query in the [Event Management Explorer][2] by selecting the `</>` icon. Filters let you limit what kinds of events a pipeline applies to. **Note**: The pipeline filtering is applied before any of the pipeline's processors. You cannot filter on an attribute that is extracted in the pipeline itself.
```
Comment (maycmlee) [inline content/en/service_management/events/pipelines_and_processors/_index.md:32]:
```suggestion
1. Choose a filter from the dropdown menu or create your own filter query in the [Event Management Explorer][2] by selecting the `</>` icon. Use filters to apply pipeline processors to specific events. **Note**: The pipeline filtering is applied before any of the pipeline's processors. You cannot filter on an attribute that is extracted in the pipeline itself.
```

File: content/en/service_management/events/pipelines_and_processors/_index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (maycmlee) [reply content/en/service_management/events/pipelines_and_processors/_index.md]: Yes, that sounds good! I edited it a little in the suggestion below, so it's: "Use the filter to apply pipelines processors to specific events."

**Review by maycmlee — APPROVED** [review]
I added a suggestion to that last bit we were talking about, but am approving so you can merge.

**Pull request merged**